lib/src/components/NxCloseButton/NxCloseButton.tsx

```diff
@@ -16,7 +16,12 @@ const NxCloseButton = forwardRef<HTMLButtonElement, ButtonHTMLAttributes<HTMLBut
       const btnClasses = classnames('nx-btn--close', className);
 
       return (
-        <NxButton ref={ref} type="button" className={btnClasses} variant="icon-only" { ...otherProps }>
+        <NxButton ref={ref}
+                  type="button"
+                  className={btnClasses}
+                  variant="icon-only"
+                  { ...otherProps }
+                  aria-label="Dismiss">
```

Comment:
By putting this last, you prevent `otherProps` from overriding it.  Is that desired?

Reply:
> By putting this last, you prevent `otherProps` from overriding it. Is that desired?

I didn't know that the order mattered, changed here: https://github.com/sonatype/sonatype-react-shared-components/pull/209/commits/253dab64fe05eb1007dba216df219445e66bacb8